### ISSUE TITLE
Larger discretization size required for ode/Rectifier example in v5

### ISSUE THREAD

**Issue by gradywright** [chebfun/chebfun#684]
When running the `alpha=40` case in the [ode/Rectifier](https://github.com/chebfun/examples/blob/master/ode/Rectifier.m) example, I found that it was necessary to increase the max discretization to 2896 to compute the solution.  In v4, the solution could be computed using on a max discretization of 2048 (as shown [here](http://www2.maths.ox.ac.uk/chebfun/examples/ode/html/Rectifier.shtml)).   

The lengths of the corresponding solutions are, however, quite similar.  In v5 the length is 2103, while in v4 the length is 2024 (note the example on the chebfun webpage is not up-to-date with the latest v4 version).   Strangely, the chebpolyplot's of the v4 and v5 solution are quite different.
![rectifierchebpolyplotalpha40v4](https://cloud.githubusercontent.com/assets/6304453/3146376/047e5fda-ea46-11e3-8607-d06b619da129.png)

![rectifierchebpolyplotalpha40](https://cloud.githubusercontent.com/assets/6304453/3146282/a7a8474a-ea44-11e3-997f-46d2393fa340.png)

**Comment by aaustin141** [comment]
With #1562 merged, I now get the coefficient plot for `v_40` has length 865, and its coefficient plot looks like this:
![684](https://cloud.githubusercontent.com/assets/3380006/8991426/6a95290a-36f0-11e5-9461-6781ec4aa823.png)
Can we now mark this issue as closed and put this example up on the website?

**Comment by trefethen** [comment]
Very nice.

**Comment by asgeirbirkis** [comment]
I agree with @aaustin141 , I think we can regard this issue as being sorted and should reintroduce the example on the website.
